main.py

```diff
@@ -1 +1,25 @@
 #Mad Libs Generator Project
+#Mad Libs Generator Project
+
+#Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+#All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
```

Comment:
Maybe we can combine noun, second_noun, and third_noun into a function since the print statement is returning the same thing for each.